packages/agent-toolkit/README.md

```diff
@@ -0,0 +1,231 @@
+# Recall Agent Toolkit
+
+> A collection of framework-agnostic agent tools for building Recall agents.
+
+## Table of Contents
+
+- [Background](#background)
+- [Usage](#usage)
+  - [Installation](#installation)
+  - [Adapters](#adapters)
+  - [Resources \& permissions](#resources--permissions)
+  - [Shared components](#shared-components)
+    - [Tools \& prompts](#tools--prompts)
+  - [Examples](#examples)
+- [Development](#development)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Background
+
+The `@recallnet/agent-toolkit` is a collection of tools for building agents that interact with Recall. It is **_heavily_** inspired by the [Stripe Agent Toolkit](https://github.com/stripe/agent-toolkit) project and makes it easy to use a unified API across different frameworks. There are framework agnostic adapters for:
+
+| Framework                                                               | Package import path                  |
+| ----------------------------------------------------------------------- | ------------------------------------ |
+| [Model Context Protocol (MCP)](https://github.com/modelcontextprotocol) | `@recallnet/agent-toolkit/mcp`       |
+| [LangChain](https://github.com/langchain-ai/langchainjs)                | `@recallnet/agent-toolkit/langchain` |
+| [OpenAI](https://github.com/openai/openai-node)                         | `@recallnet/agent-toolkit/openai`    |
+| [AI SDK](https://github.com/vercel/ai)                                  | `@recallnet/agent-toolkit/ai-sdk`    |
+
+## Usage
+
+### Installation
+
+Install the package:
+
+```bash
+npm install @recallnet/agent-toolkit
```

Comment:
Do we want to also have pnpm install steps? They aren't really all that different. Additionally, does this work with deno? Probably would via `deno add npm:@recallnet/agent-toolkit`..?